GliaWidgets/SwiftUI/Components/BackgroundSwiftUI.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+struct BackgroundSwiftUI: View {
```

Comment:
I heard you have had a meeting about SwiftUI, did you guys @gersonnoboa  @rasmustautsglia, @igorkravchenko make some decisions about naming also?
I would not like to use `SwiftUI` suffix for all View if it possible, however, would like to know about your decision also.

Reply:
Good point. We didn't touch the naming but you are making a good point here. However, some views cannot be declared without that suffix, such as Header, ActionButton, and HeaderButton because the namespace is already taken by UIKit analogs. 

fixed BackgroundSwiftUI -> Background